src/serde/value/borrowed/de.rs

```diff
@@ -537,8 +551,7 @@ impl<'de> VariantAccess<'de> for VariantDeserializer<'de> {
                 if v.is_empty() {
                     visitor.visit_unit()
                 } else {
-                    visitor.visit_seq(Array(v.iter()))
-                    // visit_array_ref(v, visitor)
+                    visitor.visit_seq(Array(v.into_iter()))
                 }
             }
             // FIXME
```

Comment:
Is this vestigial? Perhaps we can just remove the FIXME comment here

Reply:
the `FIXME` here is for the error handling (or lack thereof)